### ISSUE TITLE
[Customer] Can't locate Getopt /Std.pm during genesis discovery

### ISSUE THREAD

**Issue by cxhong** [xcat2/xcat-core#2475]
Lab Service reported this error during genesis discovery:
`````
<166>Feb 10 11:41:06 xcat.genesis.doxcat: Starting ntpd...
<166>Feb 10 11:41:06 xcat.genesis.doxcat: Waiting for ntpd to synchronize...
Can't locate Getopt/Std.pm in @INC (you may need to install the Getopt::Std module) (@INC contains: /usr/l
ocal/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib
64/perl5 /usr/share/perl5 .) at /usr/sbin/ntp-wait line 5.
BEGIN failed--compilation aborted at /usr/sbin/ntp-wait line 5.
<166>Feb 10 11:41:06 xcat.genesis.doxcat: ... ntpd did not synchronize.
<166>Feb 10 11:41:06 xcat.genesis.doxcat: Attempting to sync hardware clock...
`````
Here is xCAT version on their system:
```
[root@smn dev]# lsxcatd -a
Version 2.13.1 (git commit 62bd7a4072bb629ede16974272c993f4e707765f, built Wed Jan 11 08:45:41 EST 2017)
This is a Management Node
dbengine=SQLite
`````

**Comment by immarvin** [comment]
hi @zet809 , would you please take a look at this?

**Comment by whowutwut** [comment]
Assigned to @neo954  This issue occurs when running code changed by PR https://github.com/xcat2/xcat-core/pull/2333

**Comment by zet809** [comment]
I need to rebuild xcat-genesis-base to include those libraries.

**Comment by whowutwut** [comment]
Is this something that we have looked into potentially using containers to help simplify the re-build process of the dependencies?

**Comment by zet809** [comment]
No, for genesis-base-ppc64, I was using my CN which had been installed Fedora23 before.

**Comment by neo954** [comment]
Tested `xCAT-genesis-base-ppc64-2.13.2-snap201702150049.noarch` on a PowerVM LPAR. And get the following error message 

    [xCAT Genesis running on (none) /]# ntp-wait                                   Unexpected first line <System error>

**Comment by neo954** [comment]
Anyway, the `ntp-wait` can be run in the genesis now. We still need to solve the network problem for PowerVM LPAR, which need linux kernel module `ibmveth`. It is another problem.

**Comment by neo954** [comment]
@zet809, Please continue working on rebuild `xCAT-genesis-base-ppc64` and `xCAT-genesis-base-x86_64` with ...

  * Use CentOS 7.3 as the base Linux distro, and
  * Include all the Linux kernel modules that CentOS 7.3 shipped.

**Comment by whowutwut** [comment]
@zet809 @neo954 we have issue #2258 to track this.    If we think it's higher priority, we should probably move it up in the backlog and see if someone can work on it.

**Comment by zet809** [comment]
The whole perl library had been included in xcat-genesis-base-2.13.2 build, @neo954 If you have verified, will you pls close this defect? Thx!

**Comment by zet809** [comment]
This issue can be fixed with the genesis-base mentioned above, I had verified it, so close this defect.
